microprofile.jdt/com.redhat.microprofile.jdt.core/src/main/java/com/redhat/microprofile/jdt/internal/core/project/JDTMicroProfileProject.java

```diff
@@ -0,0 +1,68 @@
+/*******************************************************************************
+* Copyright (c) 2020 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.microprofile.jdt.internal.core.project;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.jdt.core.IJavaProject;
+
+/**
+ * JDT MicroProfile project.
+ * 
+ * @author Angelo ZERR
+ *
+ */
+public class JDTMicroProfileProject {
+
+	public static final String MICROPROFILE_CONFIG_PROPERTIES_FILE = "META-INF/microprofile-config.properties";
+	public static final String APPLICATION_PROPERTIES_FILE = "application.properties";
+
+	private final List<IConfigSource> configSources;
+
+	private static final String QUARKUS_HTTP_PORT = "quarkus.http.port";

```

Comment:
will need to be decoupled eventually, as it's a quarkus thing, not an MP thing

Reply:
I agree we should manage that with new extension point (like we have done for properties provider), but I think we should create a new issue for that which is not trivial.